.ddev/commands/web/install

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
This could probably use a more descriptive name, like `drupal-install` since it's that...